scenario/consistency_checker.py

```diff
@@ -326,10 +327,20 @@ def check_storages_consistency(
     return Results(errors, [])
 
 
+def _is_secret_identifier(value):
```

Comment:
No type annotation? Do we have type checking turned on in this repo? I thought Pyright complained loudly if you missed these, or is that only in strict mode?

Reply:
```suggestion
def _is_secret_identifier(value: str):
```